src/ripple/app/ledger/Ledger.cpp

```diff
@@ -293,29 +290,28 @@ Ledger::Ledger (Ledger const& prevLedger,
     }
 }
 
-Ledger::Ledger (
-        LedgerInfo const& info,
-        Config const& config,
-        Family& family)
-    : mImmutable (true)
-    , txMap_ (std::make_shared <SHAMap> (SHAMapType::TRANSACTION,
-        info.txHash, family))
-    , stateMap_ (std::make_shared <SHAMap> (SHAMapType::STATE,
-        info.accountHash, family))
+Ledger::Ledger(LedgerInfo const& info, Config const& config, Family& family)
+    : mMutable(false)
+    , txMap_(std::make_shared<SHAMap>(
+          SHAMapType::TRANSACTION,
+          info.txHash,
+          family))
+    , stateMap_(
+          std::make_shared<SHAMap>(SHAMapType::STATE, info.accountHash, family))
     , rules_{config.features}
-    , info_ (info)
+    , info_(info)
 {
     info_.hash = calculateLedgerHash (info_);
 }
 
-Ledger::Ledger (std::uint32_t ledgerSeq,
-        NetClock::time_point closeTime, Config const& config,
-            Family& family)
-    : mImmutable (false)
-    , txMap_ (std::make_shared <SHAMap> (
-          SHAMapType::TRANSACTION, family))
-    , stateMap_ (std::make_shared <SHAMap> (
-          SHAMapType::STATE, family))
+Ledger::Ledger(
+    std::uint32_t ledgerSeq,
```

Comment:
Not actionable here, but just a comment for you to think about: if possible, I'd love to see a `clang-format` configuration that either indents the arguments or the constructor initializers one extra level, to visually distinguish argument lists from constructor initializers.